src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -5,6 +5,7 @@
 import os
 
 NODE_VERSION_DEFAULT = "10.14"
+NODE_EXACT_VERSION_DEFAULT = "10.14.1"
```

Comment:
What is the purpose of the Min version here?

Reply:
Just to differentiate between the two values, `10.14` and `10.14.1`. The runtime value is `node|10.14`, but the webapp config `WEBSITE_NODE_DEFAULT_VERSION` needs to be set to `10.14.1` rather than `10.14`